### PR TITLE
fix(wedlocks, unlock-app): email fixes

### PR DESCRIPTION
**Pull request by julien51**
# Description

@kalidiagne adding more fixes for email sending.

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/interface/locks/Settings/elements/EmailTemplatePreview.tsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ export const EmailTemplatePreview = ({
 
     const params = {
       lockName: 'Email Preview',
-      keychainUrl: 'https://app.unlock-protocol.com/keychain',
+      keychainUrl: 'https://app.unlock-protocol.com/keychain', // TOFIX @kali: never use hardcoded URLs.
```
Comment (julien51) [inline unlock-app/src/components/interface/locks/Settings/elements/EmailTemplatePreview.tsx:97]: @kalidiagne Please fix this ASAP

File: unlock-app/src/components/interface/locks/Settings/elements/EmailTemplatePreview.tsx [hunk 2]
```diff
@@ -108,19 +108,29 @@ export const EmailTemplatePreview = ({
       eventTime: '{Event time}',
       eventAddress: '{Event address }',
       // certificate details
-      certificationDetail: 'https://example.it',
+      certificationDetail: 'https://example.it', // TOFIX @kali: What is this?
```
Comment (julien51) [inline unlock-app/src/components/interface/locks/Settings/elements/EmailTemplatePreview.tsx:111]: Same here!

File: wedlocks/config-builder.js [hunk 3]
```diff
@@ -50,6 +50,6 @@ console.log(
     unlockEnv,
     wedlocksPrivateKey,
     sender:
-      process.env.SMTP_FROM_ADDRESS || 'Unlock Labs <hello@unlock-protocol.com>', // TODO: can we do better eventually?
+      process.env.SMTP_FROM_ADDRESS || 'hello@unlock-protocol.com',
```
Comment (julien51) [inline wedlocks/config-builder.js:53]: I think this is what was creating the errors you were seeing.

**Review by julien51 — COMMENTED** [review]
@kalidiagne

File: unlock-app/src/components/interface/locks/Settings/elements/EmailTemplatePreview.tsx (hunk 2 above)
Comment (kalidiagne) [reply unlock-app/src/components/interface/locks/Settings/elements/EmailTemplatePreview.tsx:111]: this is only a placeholder, changing to keep same format as above

File: wedlocks/config-builder.js (hunk 3 above)
Comment (kalidiagne) [reply wedlocks/config-builder.js]: Is `SMTP_FROM_ADDRESS` also changed from GitHub variables?

**Review by kalidiagne — APPROVED** [review]

**Pull request merged**